src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevService.kt

```diff
@@ -110,10 +113,18 @@ class BrevService(
             }
 
         val brevmottakereFraBehandling = behandling?.let { brevmottakerService.hentBrevmottakere(it.id) } ?: emptyList()
+        val brevmottakerDtoListe = manueltBrevDto.manuelleBrevmottakere + brevmottakereFraBehandling
         val mottakere =
             brevmottakerService.lagMottakereFraBrevMottakere(
-                manueltRegistrerteMottakere = manueltBrevDto.manuelleBrevmottakere + brevmottakereFraBehandling,
+                manueltRegistrerteMottakere = brevmottakerDtoListe,
+            )
+
+        if (!BrevmottakerAdresseValidering.erBrevmottakereGyldige(brevmottakerDtoListe)) {
+            throw FunksjonellFeil(
+                melding = "Det finnes ugyldige brevmottakere i utsending av manuelt brev",
+                frontendFeilmelding = "Adressen som er lagt til manuelt har ugyldig format, og brevet kan ikke sendes. Du må legge til manuell adresse på nytt.",
```

Comment:
Ukjent med brevmottaker domenet, men er det mulig å legge til flere brevmottakere?
Bør man da referere til hvilket brevmottaker som har ugyldig adresse, så det blir lettere for SB å rette opp i?

Reply:
Vi kan legge til flere brevmottakere ja. Denne valideringen vil kunne trigges for saker som har ligget en stund der saksbehandler la til brevmottakere før vi deployet ut frontend-valideringen vår (som skjedde i forrige uke). Vi burde i fremtiden ikke komme i slike states, men akkurat nå i denne mellomperioden er det et problem, spesielt for EØS-saker som kan ligge under arbeid en stund. Siden dette ikke vil skje ofte fremover tror jeg ikke at det er verdt å endre på feilmeldingen nå. Vi må få ut denne endringen før Dokdist kan gjøre en endring i valideringen på sin side. ☺️